src/psyclone/psyir/transformations/omp_target_trans.py

```diff
@@ -158,11 +158,12 @@ def validate(self, node, options=None):
         :raises TransformationError: if its a function and the target region
             attempts to enclose the assingment setting the return value.
         '''
+        device_string = options.get("device_string", "") if options else ""
```

Comment:
Please document this new options argument and mention it in the apply docstring too. How much work would it be to convert this transformation to use the new kwargs form for options?

Reply:
Added the docstring. Converting to kwargs got quite complicated so I would prefer to do it separately.